File: consumer-java/src/main/java/consumer/model/lead/status/LeadStatusRequest.java
package consumer.model.lead.status;

import java.util.List;

public class LeadStatusRequest {
    List<String> referralIds;

    public List<String> getReferralIds() {
        return referralIds;
    }

    public void setReferralIds(List<String> referralIds) {
        this.referralIds = referralIds;
    }
}

File: consumer-python/model/LeadStatusRequest.py
import json


class LeadStatusRequest:
    def __init__(self):
        pass

    referralIds = []

    def setReferralIds(self, value):
        self.referralIds = value

    def __str__(self):
        return '{\"referralIds\": ' + json.dumps(self.referralIds) + \
               '}'